indico/modules/rb/notifications/notifications_test.py

```diff
@@ -0,0 +1,130 @@
+    dummy_reservation.end_dt = datetime(2022, 11, 11, 14, 37)
+    _assert_snapshot(snapshot, snapshot_name, 'reservations',
+                     reservation=dummy_reservation, reason='Because I want to.')
+
+
+@pytest.mark.parametrize('snapshot_name', (
+    'creation_email_to_manager.txt', 'creation_email_to_user.txt',
+    'creation_pre_email_to_manager.txt', 'creation_pre_email_to_user.txt',
+))
+def test_reservation_creation_emails_plaintext(snapshot, snapshot_name, create_reservation):
+    res = create_reservation(id=1337)
+    res.start_dt = datetime(2022, 11, 11, 13, 37)
+    res.end_dt = datetime(2022, 11, 11, 14, 37)
+    _assert_snapshot(snapshot, snapshot_name, 'reservations', reservation=res,
+                     RepeatMapping=RepeatMapping, RepeatFrequency=RepeatFrequency)
+
+
+@pytest.mark.parametrize('snapshot_name', (
+    'modification_email_to_manager.txt', 'modification_email_to_user.txt'
+))
+def test_reservation_modification_emails_plaintext(snapshot, snapshot_name, dummy_reservation):
+    dummy_reservation.id = 0
+    dummy_reservation.start_dt = datetime(2022, 11, 11, 13, 37)
+    dummy_reservation.end_dt = datetime(2022, 11, 11, 14, 37)
+    _assert_snapshot(snapshot, snapshot_name, 'reservations', reservation=dummy_reservation)
+
+
+@pytest.mark.parametrize('snapshot_name', (
+    'occurrence_cancellation_email_to_manager.txt', 'occurrence_cancellation_email_to_user.txt',
+    'occurrence_rejection_email_to_manager.txt', 'occurrence_rejection_email_to_user.txt'
+))
+def test_reservation_occurrence_emails_plaintext(snapshot, snapshot_name, dummy_reservation):
+    dummy_reservation.id = 0
+    dummy_reservation.start_dt = datetime(2022, 11, 11, 13, 37)
+    dummy_reservation.end_dt = datetime(2022, 11, 12, 14, 37)
+    occurrence = {'start_dt': datetime(2022, 11, 11, 13, 37), 'rejection_reason': 'A valid reason!'}
+    _assert_snapshot(snapshot, snapshot_name, 'reservations', reservation=dummy_reservation, occurrence=occurrence)
+
+
+@pytest.mark.parametrize('snapshot_name', (
+    'rejection_email_to_manager.txt', 'rejection_email_to_user.txt'
+))
+def test_reservation_rejection_emails_plaintext(snapshot, snapshot_name, dummy_reservation):
+    dummy_reservation.id = 0
+    dummy_reservation.start_dt = datetime(2022, 11, 11, 13, 37)
+    dummy_reservation.end_dt = datetime(2022, 11, 11, 14, 37)
+    _assert_snapshot(snapshot, snapshot_name, 'reservations',
+                     reservation=dummy_reservation, reason='Because I want to.')
+
+
+@pytest.mark.parametrize('snapshot_name', (
+    'reservation_info.txt', 'reservation_key_info.txt'
+))
+def test_reservation_emails_plaintext(snapshot, snapshot_name, dummy_reservation):
+    dummy_reservation.id = 0
+    dummy_reservation.room.key_location = 'In a pocket dimension reachable via interpretative dance.'
+    dummy_reservation.start_dt = datetime(2022, 11, 11, 13, 37)
+    dummy_reservation.end_dt = datetime(2022, 11, 11, 14, 37)
```

Comment:
With that many changes I'd use the `create_reservation` fixture and just pass the data to it as kwargs (same in similar places)

(I think this can/should be done especially in places where the ID gets changed manually)